widgets/src/main/java/org/pentaho/gwt/widgets/client/utils/TimeUtil.java

```diff
@@ -132,6 +132,49 @@ public int getPrevious() {
     }
   } /* end enum */
 
+  /**
+   * Names of enum are used as key in resource bundle.
+   */
```

Comment:
Does this comment also apply to this enum? Could not find any new resource bundle strings with these names.

Is there an unlinked associated PR?

Reply:
It looks like there is one: https://github.com/pentaho/pentaho-platform/pull/5380.

Ideally, please link associated PRs to each other (or have one of the PRs be the head of a set, that links to the other PRs).

Still, the first statement applies.